sdks/python/generate_pydocs.sh

```diff
@@ -0,0 +1,79 @@
+#!/bin/bash
```

Comment:
bind the script to the maven `package` lifecycle to make sure it doesn't break and can simplify the instructions, and make it platform-specific. (See what archetypes do.)